src/serviceBundle/ServiceFactory.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace peter\components\serviceBundle;
+
+use peter\components\serviceBundle\Services\Mailgun;
+use peter\components\serviceBundle\Services\Imgur;
+use peter\components\serviceBundle\Services\ImageShack;
+use peter\components\serviceBundle\Services\Bitly;
+use peter\components\serviceBundle\Services\McAfee;
+use peter\components\serviceBundle\Services\Google;
+use peter\components\serviceBundle\Services\SendGrid;
+
+class ServiceFactory
+{
+    public function create($service) 
+    {
+        switch ($service) {
```

Comment:
What do you think use the following sample code to replace the switch....case?

$service = 'MailGun';
$object = new $service();

Reply:
Yes. Let me look at the options for replacing the switch statement.